phpunit/functional/Glpi/Inventory/Assets/NetworkPortTest.php

```diff
@@ -1531,4 +1531,130 @@ public function testNetworkEquipmentsConnectionsConverted(): void
         //8 connections on port, but one related to existing equipment
         $this->assertCount(7, $unmanaged->find());
     }
+
+    public function testDefaultInstantiationType()
+    {
+        global $DB;
+        $json_str = '
+        {
+            "action": "inventory",
+            "content": {
+                "accesslog": {
+                  "logdate": "2023-12-11 09:39:16"
+                },
+                "bios": {
+                  "assettag": "Asset-1234567890",
+                  "bdate": "2013-10-29",
+                  "bmanufacturer": "American Megatrends Inc.",
+                  "bversion": "1602",
+                  "mmanufacturer": "ASUSTeK COMPUTER INC.",
+                  "mmodel": "Z87-A",
+                  "msn": "131219362301208",
+                  "skunumber": "All",
+                  "smanufacturer": "ASUS",
+                  "smodel": "All Series"
+                },
+                "hardware": {
+                  "chassis_type": "Desktop",
+                  "datelastloggeduser": "Mon Dec 11 09:34",
+                  "defaultgateway": "192.168.1.1",
+                  "dns": "127.0.0.53",
+                  "lastloggeduser": "teclib",
+                  "memory": 32030,
+                  "name": "teclib-asus-desktop",
+                  "swap": 2047,
+                  "uuid": "31042c80-d7da-11dd-93d0-bcee7b8de946",
+                  "vmsystem": "Physical",
+                  "workgroup": "home"
+                },
+                "networks": [
```

Comment:
Maybe you can add another network port without MAC address, and make sure only the correct one has been created.

Reply:
With that test, this will be OK for me :)

Reply:
done =)